can-observation.js

```diff
@@ -101,6 +101,11 @@ function Observation(func, context, compute){
 	this.childDepths = {};
 	this.ignore = 0;
 	this.needsUpdate= false;
+	if (!func) {
+		this.start = function(){
```

Comment:
`start` is needed because it is called in `update()`, and if I modified update to not call `start()`, I wasn't getting updates. There may be a better way to do this though, I am open to suggestion.

I'll declare the function outside of the constructor and reuse it.